planner/cascades/testdata/stringer_suite_out.json

```diff
@@ -285,6 +285,21 @@
           "Group#16 Schema:[test.t.b]",
           "    TableScan_24 table:t"
         ]
+      },
+      {
+        "SQL": "select b, f, g from t where t.b = 2 and t.f = 1 order by t.f, t.g",
+        "Result": [
+          "Group#0 Schema:[test.t.b,test.t.f,test.t.g], UniqueKey:[test.t.f,test.t.f,test.t.g]",
+          "    Sort_4 input:[Group#1], test.t.f:asc, test.t.g:asc",
+          "Group#1 Schema:[test.t.b,test.t.f,test.t.g], UniqueKey:[test.t.f,test.t.f,test.t.g]",
+          "    Projection_3 input:[Group#2], test.t.b, test.t.f, test.t.g",
+          "Group#2 Schema:[test.t.b,test.t.f,test.t.g], UniqueKey:[test.t.f,test.t.f,test.t.g]",
+          "    TiKVSingleGather_6 input:[Group#3], table:t",
+          "Group#3 Schema:[test.t.b,test.t.f,test.t.g], UniqueKey:[test.t.f,test.t.f,test.t.g]",
+          "    Selection_16 input:[Group#4], eq(test.t.b, 2), eq(test.t.f, 1)",
+          "Group#4 Schema:[test.t.b,test.t.f,test.t.g], UniqueKey:[test.t.f,test.t.f,test.t.g]",
+          "    TableScan_5 table:t"
```

Comment:
Maybe the cost let the SQL can't use `TiKVDoubleGather`.

Reply:
We didn't use the cose model in this test. It must be related to `Covert2Gathers()`.

Reply:
Yeah, that's right.